src/vs/workbench/parts/output/browser/output.contribution.ts

```diff
@@ -45,3 +29,25 @@ registerMode({
 	moduleId: 'vs/workbench/parts/output/common/outputMode',
 	ctorName: 'OutputMode'
 });
+
+// Register Output Panel
+(<panel.PanelRegistry>platform.Registry.as(panel.Extensions.Panels)).registerPanel(new panel.PanelDescriptor(
```

Comment:
@bpasero for some reason it is working wihtout it also. I can add it for precaution